keadm/cmd/keadm/app/cmd/util/k8sinstaller.go

```diff
@@ -82,10 +82,17 @@ func installCRDs(kubeConfig, master string) error {
 	}
 	for _, crdFile := range []string{"devices/devices_v1alpha1_device.yaml",
 		"devices/devices_v1alpha1_devicemodel.yaml"} {
-		//Download the tar from repo
-		dwnldURL := fmt.Sprintf("cd %s && wget -k --no-check-certificate --progress=bar:force %s/%s", KubeEdgeCrdPath+"/devices", KubeEdgeCRDDownloadURL, crdFile)
-		_, err := runCommandWithShell(dwnldURL)
+		//check it first, do not download when it exists
+		_, err := os.Lstat(KubeEdgeCrdPath + "/" + crdFile)
 		if err != nil {
+			if os.IsNotExist(err) {
+				//Download the tar from repo
+				dwnldURL := fmt.Sprintf("cd %s && wget -k --no-check-certificate --progress=bar:force %s/%s", KubeEdgeCrdPath+"/devices", KubeEdgeCRDDownloadURL, crdFile)
+				_, err := runCommandWithShell(dwnldURL)
+				if err != nil {
+					return err
+				}
+			}
 			return err
```

Comment:
Need else here, or it will return the err after download the crd :)

Reply:
My mistake, have tested it.